### DISCUSSION TITLE
绳网能发语音了大家快来听听

### DISCUSSION THREAD

**Discussion by homteam** [share121/inter-knot#2676] [其他]
## 语音太好用了👍
[ı||ııı||ı 5”](https://www.bilibili.com/video/BV1Uiebe6Eqk?vd_source=be2f043db652c32f9b6a949e46335c98)

**Comment by ysd1911** [comment]
警惕绳网新型诈骗！

**Comment by qqqqqq77777** [comment]
可恶，又被骗了

**Comment by LeiMoo** [comment]
跑这来了，也躲不掉是吧

**Comment by MaskedFools-Mochun** [comment]
诈骗犯？何时来的

**Comment by Chihaya-Yuka** [comment]
诈骗犯？何时来的

**Comment by SKAIZH** [comment]
我是治安官朱鸢，你被逮捕了！

**Reply by homteam** [reply]
对不起警官，为了赎罪我可以帮你洗脚🥵

**Comment by GGGGG666654** [comment]
每日上一当，当当不一样

**Comment by Rolesican** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" >！

**Comment by nninng** [comment]
警惕新型网络诈骗

**Comment by WXZ1025** [comment]
我已经报警了，你这种人马上就会被抓了😡😡😡

**Reply by homteam** [reply]
警官快用皮鞭惩罚我这个诈骗犯吧🥵

**Comment by Maple013** [comment]
滴嘟——
前新艾利都治安局治安官提醒您，保护个人隐私，谨防信息泄露，请安全使用绳网，不要轻易点击其他人发布的网页链接，以防被[Never Gonna Give You Up](https://www.bilibili.com/video/BV1z34y1C7Kc/?p=1)！

**Reply by homteam** [reply]
可恶居然也被诈骗了😡

**Comment by gebulinjisi** [comment]
又被骗了<img src="https://img-static.mihoyo.com/communityweb/upload/12a80a8f5da3949b086c497b8cc4b5d3.png" alt="安比_头痛" width="75" >

**Comment by AnonCN** [comment]
躲不掉啊

**Comment by aizai1173** [comment]
到这来也要被骗<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by lllliufeng** [comment]
我是防骗小助手，我是不是来晚了（

**Comment by flowingwind1210** [comment]
次次看，次次被骗<img src="https://bbs-static.miyoushe.com/static/2023/07/17/9eec8f933aa99e2e6a3160e861fa2e63_8010281769707476075.png" alt="流浪者-开小差" width="75" >
